lib/large_object_store.rb

```diff
@@ -119,7 +119,7 @@ def serialize(value, **options)
 
       if compress?(value, **options)
         flag |= COMPRESSED
-        value = Zlib::Deflate.deflate(value)
+        value = Zstd.compress(value, 3)
```

Comment:
I see that this argument is `compression_level` but it might be good to have a constant here and maybe an comment explaining of why we use the level we use